jscomp/core/design.md

```diff
@@ -134,11 +134,11 @@ Lalias-bound variables are never assigned, so it can only
 appear in `Lvar`, then it is easy to eliminate it 
 
 
-# interaction between `bs.splice` and `|>`
+# interaction between `variadic` and `|>`
```

Comment:
Are these interchangeable? Never thought of it myself.

Reply:
Yes. `bs.splice` is the old name for (bs.)variadic.